.circleci/config.yml

```diff
@@ -481,6 +481,7 @@ jobs:
           echo ${COMMAND} > ./command.sh && unbuffer bash ./command.sh | ts
 
           retrieve_test_reports
+          docker stats --all --no-stream
```

Comment:
This appears to only render a snapshot of usage _after_ all tests have finished, which is not very useful. In this case, it's printing 0% CPU for all tests...

One method that might work better is to check these pseudo-files for active CPU time since boot: https://www.datadoghq.com/blog/how-to-collect-docker-metrics/#cpu-pseudo-files

I hope that we would not need to continuously run and manage a daemon like cAdvisor for this, but I'm not really seeing any easy way to aggregate the resource utilization over the lifetime of the job...

Reply:
That said, the net I/O and block I/O printed are cumulative, so I suppose it's worth keeping this around just for that.

Reply:
@yns88 thank you for the feedback, I will look into a more comprehensive CPU utilization tool in a separate PR.